### PR TITLE
Add multi input file type support

### PR DESCRIPTION
**Pull request by KeisukeYamashita**
## What

This pr implements multiple input type. For example, 

```
$ conftest test config.yml traefik.toml gke.tf -p /path/to/my/common/policy.rego
```

## Why

* Address https://github.com/instrumenta/conftest/issues/158
* In organizations, there are rules that don't depend on the file type.
    * For better policy abstraction, I thought multiple input type will be great.
* Before now, if you pass multiple inputs, it will try to parse with the parser associated with the first filetype as reported in the issue.

### CODE REVIEW

File: parser/config_test.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+				"../examples/terraform/gke.tf",
+				"../examples/edn/sample_config.edn",
+			},
+		},
+	}
+
+	for n, test := range testTabel {
+		t.Run(n, func(t *testing.T) {
+			ctx := context.Background()
+			_, err := GetConfigurations(ctx, "", test.fileList)
```
Comment (KeisukeYamashita) [inline parser/config_test.go]: I didn't check the configurations because it is hard to look inside. This can be covered in the acceptance tests.

**Comment by KeisukeYamashita** [comment]
Thanks for your feedback. I will love to work on that

File: README.md [hunk 2]
```diff
@@ -167,6 +167,14 @@ As of today `conftest` supports the following output types:
 - [TAP](https://testanything.org/): `--output=tap`
 - Table `--output=table`
 
+## Multi input type
+
+Conftest supports different types of input file type.
```
[the remark this answers is not in the document]
Comment (KeisukeYamashita) [reply README.md]: Thank you, I've added a small section in there.

File: README.md [hunk 3]
```diff
@@ -92,6 +92,15 @@ The `--input` flag can also be a good way to see how different input types would
 conftest parse examples/hcl2/terraform.tf -i hcl2
 ```
 
+#### Multi input type
+
+Conftest supports different types of input file type in a single call.
```
Comment (Blokje5) [inline README.md]: Maybe it should be slightly reworded: `` `conftest` supports multiple different input types in a single call:``. The second line is already implied by the `--input` section above.

Comment (KeisukeYamashita) [reply README.md]: Thanks for your advice.

**Pull request merged**